microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,88 @@
+
+## Opportunity
+In order to better reflect a customer’s regional preferences (especially in the cases where the customer has changed their default preferences in the OS), Microsoft Edge is making a new Setting available in the “Language” section of Settings:
+
+![Screenshot of Edge setting UI titled "Share additional OS regional format" with the option "limited (default)" selected.](media/os-regional-setting-screenshot.msft.png)
+
+This setting allows the customer’s OS regional preferences to be automatically made available to websites under “limited” cases by default (more on that below), and offers two other options, including “never” which reverts the feature back to legacy behavior. 
+
+This additional sharing will allow websites to have visibility into more customer’s specific regional preferences than was previously possible. Customers that changed their OS regional preferences may now see their preferences reflected automatically in more websites. Previously, these users preferences would not be visible to websites, and their [incorrect] preferences would be assumed from the browser’s display language default region instead. 
+
+|    | What Setting Edge uses? |
+|----|-------------------------|
+| Language/region used for formatting date, time, numbers, etc. | ~~The language in the edge://settings/languages list specified as the “displayed language”~~<br><mark>The language/region specified in the OS settings*</mark> |
+
+*Depending on the new “Share additional OS regional format” UI value (details below).
+
+## Potential Misuse 
+We believe that “limited” is the right default that allows the most customer’s regional preferences to be respected with the least amount of risk of the website misusing this information. To understand why these options exist, one must understand a bit about the ways in which regional information expressed by the OS can be misused by websites. 
+
+One way in which the OS regional information could be misused is to compromise the user’s privacy—the regional information would be one additional bit of fingerprinting entropy. In combination with many other preexisting bits of entropy in the platform today, the user could be uniquely identified leading to more potential secondary harms. It’s important to remember that a user’s region and language are already available via the web platform, but users generally fall into well-recognized regional codes that are similar across geographies, and because the sample sizes are so large, the risk of any specific regional code “standing out” is small and thus the risk is small. If the user changes their OS preferences to a unique language + region, websites may be able to identify the anomaly in a population, and easily uniquely identify the associated customer. 
+
```

Comment:
i'll probably invent an h4 label/heading to distinguish each long paragraph, then break up long paragraphs

Reply:
Done